octopoes/octopoes/repositories/origin_repository.py

```diff
@@ -63,6 +63,7 @@ def serialize(cls, origin: Origin) -> dict[str, Any]:
         data = origin.model_dump()
         data[cls.pk_prefix] = origin.id
         data["type"] = origin.__class__.__name__
+        data["result"] = list(dict(zip(data["result"], [None] * len(data["result"]))).keys())
```

Comment:
It took me a few times reading this and the PR title to understand it; but if I'm correct you're trying to remove duplicates in the `data["result"]` list, while also keeping the order?

If that's the case, I'd like to suggest this snippet as it's a bit shorter and also a common approach:
```suggestion
        data["result"] = list(dict.fromkeys(data["result"]))
```